src/drivers/github.js

```diff
@@ -32,6 +33,13 @@ class Github {
 
     this.repo = repo;
     this.token = token;
+    this.octokit_options = {};
+
+    if (!repo.includes('github.com')) {
+      const repo_url = new url.URL(repo);
+
+      this.octokit_options.baseUrl = 'https://' + repo_url.host + '/api/v3';
```

Comment:
We could do this inside the function that returns the octokit just only passing the repo so we dont need to create the property octokit_options?

so the call would be

```js
await octokit(this.token, this.repo)...
```

Reply:
Good suggestion! I'll push a commit shortly for this.

Reply:
Changed in https://github.com/iterative/cml/pull/331/commits/06d2e5d21c4a2235b37cfd079b849d3d4c54f0b8.